bumpCversion.py

```diff
@@ -94,127 +131,90 @@ def get_config(config_file):
     return config_file_exists, components
 
 
-def get_target_files(args):
+def get_target_files(config, component):
     """
     If a version file was specified on the CLI, use it. Otherwise, look for a configuration file.
     """
     target_files = []
-    if args.version_file:
-        target_files.append(args.version_file)
-        return target_files
-    else:
-        config_file_exists, cfg_components = get_config(args.config_file)
-
-        if not config_file_exists:
-            print("Nothing to do, configuration does not exist!")
-            return
-
-        for comp in cfg_components:
-            if args.component == comp.name:
-                print("\nUsing component:", comp.name + '\n')
-                for path in comp.paths:
-                    target_files.append(path)
 
-    return target_files
+    config_file_exists, cfg_components = get_config(config)
 
-def valid_version_congruence(args, target_files, versions):
-    """
-    Check to see if version number is the same across all files.
-    """
-    while target_files:
-        filetype = get_filetype_object(args, target_files)
-        # add version to unique set
-        versions.add(str(filetype.version_number))
-        target_files.pop(0)
+    if not config_file_exists:
+        print("Nothing to do, configuration does not exist!")
+        return
 
-    if len(versions) == 1:
-        return 1
-    else:
-        print("Multiple Verisons Found -> " + str(versions))
-        raise(VersionException(Exception))
+    for comp in cfg_components:
+        if component == comp.name:
+            print("\nUsing component:", comp.name + '\n')
+            for path in comp.paths:
+                target_files.append(path)
+
+    return target_files
 
 
-def print_dry(args, target_files, versions):
+def valid_version_congruence(config, versions):
+    """
+    Check to see if version number is the same across all files per component.
+    """
+    config_file_exists, cfg_components = get_config(config)
+    for component in cfg_components:
+        if config_file_exists:
+            while component.paths:
+                filetype = get_filetype_object(component.paths)
+                # add version to unique set
+                versions.add(str(filetype.version_number))
+                component.paths.pop(0)
+            if len(versions) == 1:
+                versions.clear()
+                continue
+            else:
+                print("Multiple Verisons Found -> " + str(versions))
+                raise(VersionException(Exception))
+
+    return 1
```

Comment:
Should this be `return True` instead of `return 1`?

Reply:
Changed